dpnp/dpnp_iface_statistics.py

```diff
@@ -301,8 +301,28 @@ def max(input, axis=None, out=None, keepdims=numpy._NoValue, initial=numpy._NoVa
     """
 
     if not use_origin_backend(input):
+        isaxis = True
```

Comment:
What cases you are trying to handle here? Is it a code to validate axis as input?
It would be better to write small comment in the code about it.
If it a user input validation procedure - it is better to put it in some "utils" place. Perhaps it is already where.

Reply:
@shssf 
DPNP dparray::__init__(): Negative values in 'shape' are not allowed

To avoid causing this error at the Cython level, this check has been added. The error call removal was requested in the PR above.